docs/index.rst

```diff
@@ -15,19 +15,27 @@ Welcome to MetalK8s's documentation!
 
 .. note: Keep this in sync with the toctree in `index-latex.rst`...
 
-.. toctree::
-   :maxdepth: 2
-   :caption: Contents:
+.. only:: not release
 
-   installation/index
-   operation/index
-   developer/index
-   glossary
+   .. toctree::
+      :maxdepth: 2
+      :caption: Contents:
 
+      installation/index
+      operation/index
+      developer/index
+      glossary
 
-Indices and tables
-==================
+      * :ref:`genindex`
+      * :ref:`modindex`
+      * :ref:`search`
```

Comment:
Do we don't include the search etc. on a release build? Kinda odd.

Reply:
I think this was a request from @lucieleonard. In the new theme, we always show the Search bar in the header. For other topics, I don't think we show them at all.

It's easy to add back if you think we need to.

Reply:
Added back (and removed `modindex` as we don't have any auto-documented modules).